src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        final String DIVIDER = "    ——————————————————————————————————————————————————————————————";
+        System.out.println(DIVIDER);
+        System.out.println("    Hello! I'm Duke\n    What can I do for you?");
+        System.out.println(DIVIDER);
+        String line;
+        String[] lists = new String[100];
+        String[] flags = new String[100];
+        String[] doneTasks = new String[100];
+        String[] dates = new String[100];
+        Arrays.fill(doneTasks," ");
+        int number = 0;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
```

Comment:
The body of each if-else statement here can be split into functions to improve the code quality.